### DISCUSSION TITLE
Running superset asynchronously in non-dev

### DISCUSSION THREAD

**Discussion by anthony6401** [apache/superset#16105] [Q&A / Help] [answered]
Hi, I have been trying to run superset asynchronously. I am able to run it on dev (by running `docker-compose up`), and it is able run async fine. However, when I start superset with `docker-compose -f docker-compose-non-dev.yml up`, it just won't work. All the queries is running synchronously.  I think I have set the config file properly, since I works fine if I run it on dev.

Any help is very appreciated regarding this matter.

**Comment by nytai** [comment, marked as answer]
the `docker-compose-non-dev.yml` file hasn't been updated with the websocket proxy service. The motivation behind the non-dev workflow is to quickly get superset up an running. Since the websocket service requires that a docker image be built it will just make the whole process much slower. You can probably just copy in the service def from the `docker-compose.yml` file and it should work.

**Reply by anthony6401** [reply]
I see. Thank you so much for your answer!
